src/OpenTelemetry/AssemblyInfo.cs

```diff
@@ -19,3 +19,7 @@
 [assembly: InternalsVisibleTo("OpenTelemetry.Tests" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2" + AssemblyInfo.MoqPublicKey)]
 [assembly: InternalsVisibleTo("Benchmarks" + AssemblyInfo.PublicKey)]
+
+// TODO: Much of the metrics SDK is currently internal. These should be removed once the public API surface area for metrics is defined.
```

Comment:
agree this is good for short term while we iron-out the public api .

Reply:
I think we can remove this now. The MetricItem and its associated classes must be public now. (Even though its highly likely they'll change, when we try for perf tweaks!)